### PR TITLE
[1LP][RFR] - Fix for SCAP testcase

### PR DESCRIPTION
**Pull request by dgaikwad**
Fixed SCAP testcase 

### PRT Run

{{pytest: ./cfme/tests/cli/test_appliance_console.py::test_appliance_console_scap ./cfme/tests/cli/test_appliance_update.py::test_update_scap_webui  -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/cli/test_appliance_console.py [hunk 1]
```diff
@@ -585,42 +579,8 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
...
-            result = elements[0].findall('./{http://checklists.nist.gov/xccdf/1.1}result')
-            if result:
-                soft_assert(result[0].text == 'pass')
-                logger.info("{}: {}".format(rule, result[0].text))
-            else:
-                logger.info("{}: no result".format(rule))
-        else:
-            logger.info("{}: rule not found".format(rule))
+    temp_appliance_preconfig.appliance_console.scap_harden_appliance()
+    assert not temp_appliance_preconfig.appliance_console.scap_check_rules()
```
Comment (jaryn) [inline cfme/tests/cli/test_appliance_console.py]: Optional:

I realize this all is my mistake -- i should have done it before, but could you please make more clear that the `scap_check_rules` returns some kind of iterable of *failures*? The best would be to rename the method to something like `scap_failures`.

Thanks

Comment (dgaikwad) [reply cfme/tests/cli/test_appliance_console.py]: @jaryn the return value of `scap_check_rules` function itself was an iterable object, so now I done the remaining changes, please review once

File: cfme/utils/appliance/console.py [hunk 2]
```diff
@@ -116,14 +116,14 @@ def scap_check_rules(self):
             if elements:
                 result = elements[0].findall('./{http://checklists.nist.gov/xccdf/1.1}result')
                 if result:
-                    if result[0].text != 'pass':
-                        rules_failures.append(rule)
+                    rules_failures.append(False) if result[0].text != 'pass' else (
```
Comment (jawatts) [inline cfme/utils/appliance/console.py]: Looks like the functionality of the method is changing. @jaryn suggested to return a iterable list of failures (which is was doing), this is now just returning true or false. The doc block just needs to be updated.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/utils/appliance/console.py (hunk 2 above)
Comment (dgaikwad) [reply cfme/utils/appliance/console.py]: @jawatts  Updated doc string

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by jawatts** [comment]
Thanks for working this @dgaikwad !

**Pull request merged**